rubycritic.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "flog", "4.3.2"
   spec.add_runtime_dependency "reek", "3.7.1"
   spec.add_runtime_dependency "parser", ">= 2.2.0", "< 3.0"
+  spec.add_runtime_dependency "colorize"
+  spec.add_runtime_dependency "tilt"
```

Comment:
I think we can implement the template scoping without Tilt by implementing some `Template`-ish class:

```ruby
Template = Struct.new(:analyzed_module, :color) do
  def render
    erb = ERB.new # ...
    erb.render(binding)
  end
end

reports = @analysed_modules.sort.map do |mod|
  Template.new(mod, RATING_TO_COLOR[mod.rating.to_s]).render
end
```

With this the ERB template will have access to a `analyzed_module` and `color` methods. What do you think?